README.md

```diff
@@ -1,12 +1,12 @@
 # looking-glass
 
-[![Build Status](https://travis-ci.com/simplifi/looking-glass.svg?branch=master)](https://travis-ci.com/simplifi/looking-glass)
+[![Build Status](https://travis-ci.com/simplifi/looking-glass.svg?branch=master)](https://travis-ci.com/simplifi/looking-glass) [![Go Report Card](https://goreportcard.com/badge/github.com/simplifi/looking-glass)](https://goreportcard.com/report/github.com/simplifi/looking-glass) [![Release](https://img.shields.io/github/release/simplifi/looking-glass.svg)](https://github.com/simplifi/looking-glass/releases/latest)
```

Comment:
something is messed up with the Release section. click the little page at the top right of the file to see what i mean

Reply:
disregard, it works now? i guess something was up with shields.io